components/script/script_thread.rs

```diff
@@ -570,6 +574,14 @@ impl ScriptThreadFactory for ScriptThread {
 }
 
 impl ScriptThread {
+    pub fn set_mutation_observer(observer: &MutationObserver) {
```

Comment:
I tried that too, but using this also gives an error:
'Use of undeclared type or module `ScriptThread`'
won't I need to import something here as well?